### ISSUE TITLE
SC2032 and SC2033 false positives when using screen -X inside a same-named function

### ISSUE THREAD

**Issue by ssokolow** [koalaman/shellcheck#197]
Because of my familiarity with screen and the late hour, It took me a while to figure out why I was getting confusing SC2032 and SC2033 errors from this code.

``` sh
title() {
    screen -X title "$@"
    echo -ne "\033_$@\033\\";
}
```

Turns out, SC2032 and SC2033 don't recognize that `-X <command>` refers to _internal_ screen commands rather than subprocesses and were only complaining because the function _happens_ to share the same name as the internal screen command.

I've worked around this using disable pragmas (I can't rename the internal command and refuse to rename my function) but, given how confused I got and how the SC2032-disabling pragma could theoretically mask real problems by being too broadly scoped, it should probably be fixed in shellcheck.
